docs/design/map/01-per-entry-stats.md

```diff
@@ -0,0 +1,50 @@
+# IMap Per Entry Stats Config Option 
+
+|ℹ️ Since: 4.2|
+|-------------|
+
+## Background
+
+### Description
+
+After implementing per config record creation, we managed to
+break tight coupling between features and record metadata.
+Based on this, now we have a chance to enable/disable
+record metadata to reduce memory usage of a record.
+
+## Design
+
+We categorize stats in two types: **map-level** and **entry-level**.
+Map level is enabled by default but entry-level is not.
+
+Map-level stats are:
+- **hits:** total hits of map
+- **lastAccessTime:** last access time to map
+- l**astUpdateTime:** last update time of map
```

Comment:
There is a typo here
l**astUpdateTime: -> **lastUpdateTime:

Reply:
fixed